src/main/proto/marathon.proto

```diff
@@ -46,6 +46,23 @@ message HealthCheckDefinition {
   optional uint32 port = 10;
 }
 
+message ReadinessCheckDefinition {
+  enum Protocol {
+    HTTP = 0;
+    HTTPS = 3;
+  }
+  optional string name = 8;
```

Comment:
Nit: I know this was a late addition, but it might be nice to make it 1 or to move it to the end.